lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,22 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "info"
+    notification_icon = "question"
+    notification_message = "Hint (1 of 2): A fruit is the fertilized ovary from a flower. Hint (2 of 2): A fruit contains seeds of the plant."
+    hint_notification = True
+%>
+
+<div class="notification ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
```

Comment:
It seems this icon could be a few different things (success, error, info, or warning, question)  Do we need accessible text to represent the icon?  If its an error icon, we will want to explicitly add SR text that says "Error" so that the learner doesn't have to rely on the content of the message to figure out the message type.